lib/credentials/shared_ini_file_credentials.js

```diff
@@ -87,6 +100,75 @@ AWS.SharedIniFileCredentials = AWS.util.inherit(AWS.Credentials, {
   /**
    * @api private
    */
+  loadRoleProfile: function loadRoleProfile(creds, callback) {
```

Comment:
Ah, I missed that `creds` was being used elsewhere, but thanks for making the change.